app/views/provider_mailer/_unsubscribe.text.erb

```diff
@@ -0,0 +1,5 @@
+# Unsubscribe
+
+You can turn off email notifications if you do not want to be told when application statuses change:
+
+[https://apply-for-teacher-training.service.gov.uk/provider/account/notification-settings](https://apply-for-teacher-training.service.gov.uk/provider/account/notification-settings)
```

Comment:
Could we make this link conditional so that it points to the right `HostingEnvironment`?